LandLord-latest/src/main/java/biz/princeps/landlord/manager/MobsManager.java

```diff
@@ -8,14 +8,16 @@
 import org.bukkit.inventory.ItemStack;
 
 import java.util.ArrayList;
-import java.util.Collection;
+import java.util.Comparator;
+import java.util.List;
 
 public class MobsManager implements IMobManager {
 
-    private static final Collection<IMob> MOBS = new ArrayList<>();
+    private static final List<IMob> MOBS = new ArrayList<>();
     final int currentMineCraftVersion;
 
     public MobsManager() {
+        // v1_17_R1 becomes 1171. Using protocol version should be better, but it is not not supported by Spigot currently.
```

Comment:
WorldEdit exposes the data version used by the platform. We could use that if we want.

Reply:
Good idea, I chose to directly use data version, didn't remember about Bukkit unsafe values :D